src/main/java/co/omise/models/Transfer.java

```diff
@@ -133,42 +142,128 @@ public void setMetadata(Map<String, Object> metadata) {
         @JsonProperty
         private Map<String, Object> metadata;
 
-        public Create amount(long amount) {
+        public CreateRequestBuilder amount(long amount) {
             this.amount = amount;
             return this;
         }
 
-        public Create recipient(String recipient) {
+        public CreateRequestBuilder recipient(String recipient) {
             this.recipient = recipient;
             return this;
         }
 
-        public Create failFast(boolean failFast) {
+        public CreateRequestBuilder failFast(boolean failFast) {
             this.failFast = failFast;
             return this;
         }
 
-        public Create metadata(Map<String, Object> metadata) {
+        public CreateRequestBuilder metadata(Map<String, Object> metadata) {
             this.metadata = metadata;
             return this;
         }
+
+        @Override
+        protected HttpUrl path() {
+            return buildUrl(Endpoint.API, "transfers");
+        }
+
+        @Override
+        protected String method() {
+            return POST;
+        }
     }
 
-    public static class Update extends Params {
+    /**
+     * The {@link RequestBuilder<Transfer>} class for updating a particular transfer.
+     */
+    public static class UpdateRequestBuilder extends RequestBuilder<Transfer> {
+        private String transferId;
         @JsonProperty
         private long amount;
         @JsonProperty
         private Map<String, Object> metadata;
 
-        public Update amount(long amount) {
+        public UpdateRequestBuilder(String transferId) {
+            this.transferId = transferId;
+        }
+
+        public UpdateRequestBuilder amount(long amount) {
             this.amount = amount;
             return this;
         }
 
-        public Update metadata(Map<String, Object> metadata) {
+        public UpdateRequestBuilder metadata(Map<String, Object> metadata) {
             this.metadata = metadata;
             return this;
         }
+
+        @Override
+        protected HttpUrl path() {
+            return buildUrl(Endpoint.API, "transfers", transferId);
+        }
+
+        @Override
+        protected String method() {
+            return PATCH;
+        }
```

Comment:
Shouldn't payload() also be overriden here?

Reply:
@Namiii Look like it worked without call `payload()`.

Reply:
@Namiii Tested, it's not work. I'll add `payload()`. 😅

Reply:
😆